Doc/library/tkinter.simpledialog.rst

```diff
@@ -0,0 +1,38 @@
+:mod:`tkinter.simpledialog` --- Tkinter simple dialogs
+======================================================
+
+.. module:: tkinter.simpledialog
+   :platform: Tk
+   :synopsis: Simple dialog windows
+
+**Source code:** :source:`Lib/tkinter/simpledialog.py`
+
+--------------
+
+The :mod:`tkinter.simpledialog` module contains the standard classes and
+functions for handling dialogs, including custom dialog windows.
+
```

Comment:
Note that this is a convenience class provided by Tkinter, rather than a wrapper around something in Tk. I would add that it provides a way to simple way to prompt the user to enter a single item via a modal dialog.

My personal opinion is these classes are good for very simple things, but once you get past that you're better off implementing them from scratch. And not even that good in terms of look and feel. My inclination would be to replace "the standard classes and functions for handling dialogs, including custom dialog windows." with "convenience classes and functions for simple modal dialogs to get a value from the user."